pillar/elastalert.sls

```diff
@@ -124,6 +124,36 @@ elastic_stack:
                       log_level: ERROR
                   - term:
                       environment.raw: mitx-production
+      - name: mitx_saml_error
+        settings:
+          name: Authentication with MIT Kerberos is currently unavailable
+          description: >-
+            Send a message anytime an error message containing No SAMLProviderData
+            found is encountered in the mitx production logs.
+          opsgenie_key: {{ opsgenie_key }}
+          opsgenie_priority: P1
+          opsgenie_alias: mitx_saml_error
+          type: frequency
+          index: logstash-*
```

Comment:
Should we segment this to logstash-mitx-production-*?

Reply:
We want the alert for QA too though, no?

Reply:
Well, we only want to get paged for production

Reply:
Also, the environment name is already being set to production at the bottom of this stanza